fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -223,7 +223,7 @@ def disable_slide_to_type
         plist_buddy_cmd = "-c \"Add :KeyboardContinuousPathEnabled bool false\""
         plist_path = File.expand_path("~/Library/Developer/CoreSimulator/Devices/#{self.udid}/data/Library/Preferences/com.apple.keyboard.ContinuousPath.plist")
 
-        Helper.backticks("#{plist_buddy} #{plist_buddy_cmd} #{plist_path}")
+        Helper.backticks("#{plist_buddy} #{plist_buddy_cmd} #{plist_path} >/dev/null 2>&1")
```

Comment:
```suggestion
        Helper.backticks("#{plist_buddy} #{plist_buddy_cmd} #{plist_path}", print: false)
```

This _should_ also work if you want to try this 😊 ☝️

Reply:
Unfortunately, after trying this, because the output is an error it is still printed 😠